Makefile

```diff
@@ -252,6 +255,10 @@ deploy_static_network_config_nodes:
 
 .PHONY: deploy_ibip
 deploy_ibip:
+ifdef ADDITIONAL_MANIFEST_DIR
+	mkdir ${INSTALL_MANIFESTS_DIR}
```

Comment:
`-p`?

Reply:
Added `rm -rf ` in front of it so that manifests from previous run would not linger there